### PR TITLE
doc: Update installation.md

### PR DESCRIPTION
**Pull request by Ileriayo**
Replace invalid GitHub url reference with the right one that points to the codebase on the master branch.

### CODE REVIEW

File: docs/install/installation.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Argo CD is running. Don't worry, without any configuration, it will not start me
     We also provide a Kustomize base in addition to the plain Kubernetes YAML
     manifests. You can use it as remote base and create overlays with your
     configuration on top of it. The remote base's URL is
-    `https://github.com/argoproj-labs/argocd-image-updater/manifests/base`
+    `https://github.com/argoproj-labs/argocd-image-updater/tree/master/manifests/base`
```
Comment (chengfang) [inline docs/install/installation.md:23]: Maybe we can use the stable version, to be consistent with the `install.yaml` later in this doc:
https://github.com/argoproj-labs/argocd-image-updater/tree/stable/manifests/base

Comment (Ileriayo) [reply docs/install/installation.md:23]: Correct!

**Review by chengfang — APPROVED** [review]

**Comment by chengfang** [comment]
Pls see comments in https://github.com/argoproj-labs/argocd-image-updater/pull/533

**Comment by chengfang** [comment]
> Pls see comments in #533

@Ileriayo Based on the discussion in #533 , do you agree this no longer needed?

**Comment by Ileriayo** [comment]
Correct! @chengfang

**Comment by pasha-codefresh** [comment]
LGTM, but i need figure out why checks are not passing first, after we can merge this

**Comment by pasha-codefresh** [comment]
@Ileriayo could you please fix DCO?

**Comment by Ileriayo** [comment]
> @Ileriayo could you please fix DCO?

Done @p

**Review by pasha-codefresh — APPROVED** [review]

**Pull request merged**